crates/uv-virtualenv/src/virtualenv.rs

```diff
@@ -131,6 +137,11 @@ pub(crate) fn create(
                     fs::create_dir_all(&location)?;
                 }
                 OnExisting::Fail => {
+                    // Refuse to remove a non-virtual environment.
+                    if !is_virtualenv {
+                        return Err(Error::NonVirtualEnvironment(location.to_path_buf()));
+                    }
```

Comment:
I'm okay with this being breaking since it was a 0.8 regression and I consider it a significant bug.